go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/go-misc/async-retry
+
+go 1.17
```

Comment:
[question]

What is the reason for not choosing go1.19?
If there is no special reason, I think it is better to use the latest Stable version.

Reply:
Because this module is used as a library, I think we should not choose 1.19.
I'm not sure, but maybe 1.17 or 1.16 is good.

Reply:
I see. I understand.
And it says in the go directive section.

> For packages within the module, the compiler rejects use of language features introduced after the version specified by the go directive

https://go.dev/ref/mod

However, I thought 1.18 was fine, since it is only up to 1.18 that it is supported.

But I don't have a strong opinion, so I'll leave it to you to decide.

Reply:
Thanks
I'll stay
I checked https://github.com/google/go-github/blob/155b546bc0c3ae6eed3a644960dddf79b86d9ad6/go.mod#L16, I notice that google also use `1.17` as their library.

Reply:
Yes.
It is inevitable that libraries created in the past are using older versions of Go, but I didn't feel the need to make the version of a newly created library less than the support version.

But I respect your decision.
Resolve this conversation.